buildspec-deploy.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.2
+
+phases:
+    install:
+      runtime-versions:
+        nodejs: 14
+        python: 3.8
+      commands:
+        - env
```

Comment:
Might reveal secrets in the log. Check this.

Reply:
Done, removed since this was mainly for debug purposes